hca/api/templates/functions.jinja

```diff
@@ -0,0 +1,19 @@
+{% for class_info in classes %}
```

Comment:
No, I'm adding that manually in `regenerate_api.py` - line 440. There's an explanation there for why I add it there instead of here. Gist of it is there are two times I'm adding using this functions template, so to keep formatting stable even though I'm adding twice, I add the imports and the scary message manually to the top of this file.